src/server/game/TypeDefinitions.js

```diff
@@ -126,6 +126,14 @@ function Game() {
      * AFK if they are currently AFK. */
     this.autoAFKResignTime = undefined;
 
+    /** Last move a draw was offered */
+    this.whiteDrawOfferMove = undefined
+    this.blackDrawOfferMove = undefined
+
+    /** Information about players offering a draw */
+    this.whiteDrawOffer = undefined
+    this.blackDrawOffer = undefined
```

Comment:
First glace, it is unclear what types of values these are supposed to hold. It says "Information about...", are they strings? Are they booleans numbers? Please describe in the jsdoc

Reply:
I'm unfamiliar with jsdoc, but I'm certain I can make something